maxim/logger/llamaindex/client.py

```
@@ -0,0 +1,244 @@
+import time
+
+from llama_index.core.agent.workflow import (
+    AgentWorkflow,
+    AgentOutput,
+    ToolCall,
+    ToolCallResult,
+    AgentInput,
+    AgentStream
+)
+
+from maxim.logger.components.generation import GenerationResult, GenerationResultChoice
```

Ruff (0.12.2)

`maxim.logger.components.generation.GenerationResult` imported but unused

Remove unused import

(F401)